modules/TableFactory/src/main/java/com/percussion/tablefactory/PSJdbcTableComponent.java

```diff
@@ -259,10 +259,7 @@ public int hashCode()
     */
    public boolean canAlter()
    {
-      return (getAction() == ACTION_CREATE ||
-         getAction() == ACTION_NONE) ||
-         getAction() == ACTION_DELETE
-         ;
+      return true;
```

Comment:
Is there no scenario where we can't alter the table?

Reply:
Returning true in base class as extended classes are taking care of correct return.